### PR TITLE
refactor(examples): add type aliases, comments, update golang code

### PR DESCRIPTION
**Pull request by vados-cosmonic**
## Feature or Problem

This commit improves the readability of the golang echo actor in-tree WIT-powered example, adding type aliases and more comments explaining the code.

### CODE REVIEW

File: examples/golang/actors/http-echo-tinygo/main.go [hunk 1]
```diff
@@ -1,26 +1,38 @@
 package main
 
 import (
-	echo "echo/gen"
-	echo_types "echo/gen"
 	"encoding/json"
 	"fmt"
+	echo "github.com/wasmcloud/wasmcloud/examples/golang/actors/echo/gen"
```
Comment (brooksmtownsend) [inline examples/golang/actors/http-echo-tinygo/main.go:6]: nit: `golang/actors/http-echo-tinygo/gen` right?

**Review by brooksmtownsend — APPROVED** [review]
Very nice! Optional nit since I'm fairly sure it doesn't affect anything.

You got me thinking with that template... we might be able to do some smart things with file paths and wadm manifests 🤔

File: examples/golang/actors/http-echo-tinygo/main.go (hunk 1 above)
Comment (vados-cosmonic) [reply examples/golang/actors/http-echo-tinygo/main.go:6]: Ah yes! I need to change the module name too -- thanks.

Comment (vados-cosmonic) [reply examples/golang/actors/http-echo-tinygo/main.go]: Fixed!

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**